lib/propcheck.ex

```diff
@@ -341,7 +341,23 @@ defmodule PropCheck do
     defmacro forall(binding, property)
     defmacro forall({op, _, [var, rawtype]}, do: prop) when op in @in_ops do
```

Comment:
What about adding an optional `option` parameter to `forall`, which is evaluated regarding verbosity? Since we are generating the environment for the property, we could handle this stuff here. Do I miss something?

Reply:
I believe it would be simpler to add capture wrapping in the property macro. There, we already have the information regarding verbosity. This does not help with calling into `quickcheck` directly (see https://github.com/alfert/propcheck/pull/48#discussion_r184658101), but maybe creating output is ok for that other case.

If capturing in the property macro is not possible, we would need to inject code into the property passed into the macro. This will definitely make the implementation more complicated, but maybe it is possible.